ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/serde/ConnectSerdeSupplier.java

```diff
@@ -125,10 +125,11 @@ private Object specToConnect(final Object spec, final Schema schema) {
         case INT32:
           return Integer.valueOf(spec.toString());
         case INT64:
+          final Long longVal = Long.valueOf(spec.toString());
           if (Timestamp.LOGICAL_NAME.equals(schema.name())) {
-            return Timestamp.toLogical(schema, (Long) spec);
+            return new java.sql.Timestamp(longVal);
           }
-          return Long.valueOf(spec.toString());
+          return longVal;
```

Comment:
Does it need changes in `connectToSpec`?

I see this code in that method:
```
case INT64:
          if (Timestamp.LOGICAL_NAME.equals(schema.name())) {
            return Timestamp.fromLogical(schema, (Date) data);
          }
```

Reply:
`connectToSpec` is used for checking the values in streams in functional tests. These values are verified against values in a json file and since json files cannot store timestamp objects, comparing the millis is the only way to check timestamp values.